app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -215,6 +211,16 @@ def finish(*_args)
 
   alias_method :abort_job, :cleanup
 
+  def cancel(*args)
+    _log.info "Job Canceling"
+    if self.state != "canceling" # ensure change of states
+      signal :cancel
+    else
+      unqueue_all_signals
+      queue_signal(:abort_job, "Job Canceled", "cancel", *args)
```

Comment:
I don't fully understand this, seems like that is what vm_scanning is calling on cancel.